src/entities/IBeacon.ts

```diff
@@ -15,7 +14,7 @@ export interface IBeacon {
   batteryExpiryDate: Date;
   lastServicedDate: Date;
   uses: IUse[];
-  owner: IOwner;
+  owners: IOwner[];
```

Comment:
Note the wireframes refer to many owners:

![image](https://user-images.githubusercontent.com/54271433/116540374-f9fac000-a8e1-11eb-9966-a3a96dd51ca3.png)

Have updated this interface to handle.  Even though we haven't modelled "many owners" in the API domain yet, I think we should still prep for the future by having an `owners` key containing an array of 1.